src/cpu/aarch64/jit_uni_batch_normalization_s8.hpp

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
```

Comment:
```suggestion
* Copyright 2021 Intel Corporation
```

Reply:
Thank you for the suggestion. I'll fix it.